packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -238,24 +293,27 @@ const Collapse = React.forwardRef(function Collapse(props, ref) {
             [isHorizontal ? 'minWidth' : 'minHeight']: collapsedSize,
             ...style,
           }}
+          styleProps={styleProps}
           ref={handleRef}
           {...childProps}
         >
-          <div
+          <CollapseWrapper
+            styleProps={styleProps}
             className={clsx(classes.wrapper, {
               [classes.horizontal]: isHorizontal,
             })}
             ref={wrapperRef}
           >
-            <div
+            <CollapseWrapperInner
+              styleProps={styleProps}
               className={clsx(classes.wrapperInner, {
```

Comment:
Let's add the horizontal class on this slot in the `useUtilityClasses` and have here only `className={classes.wrapperInner}` Same for the `wrapper`